framework/auth/views.py

```diff
@@ -134,8 +145,20 @@ def reset_password_post(uid=None, token=None):
     }
 
 
+def reset_password_institution_post(uid=None, token=None):
+    return reset_password_post(uid=uid, token=token)
```

Comment:
A similar case to `reset_password_institution_get`, there is no need to use a separate view. Here is the updated route definition.

```python
Rule(
    '/resetpassword-institution/<uid>/<token>/',
    'post',
    auth_views.reset_password_post,
    OsfWebRenderer('public/resetpassword_institution.mako', render_mako_string, trust=False)
),
```

Reply:
Same as above.